genai_bench/metrics/aggregated_metrics_collector.py

```diff
@@ -168,6 +172,21 @@ def aggregate_metrics_data(
                 if value is None:
                     logger.info(f"{i}th request has NoneType value in metric {key}.")
                     continue
+
+                # Special filtering for TPOT and inference_speed: exclude abnormal values
```

Comment:
Why do we add final filtering here? Now we are having filtering logic accross two places. It is hard to maintain.